desktop/apps/partner2/components/fixed_cells_count_carousel/view.coffee

```diff
@@ -29,8 +29,10 @@ module.exports = class FixedCellsCountCarousel extends Backbone.View
       _.map @fetchOptions, (options) =>
         @collection.fetch data: options, remove: false
     ).then (values) =>
-      objects = [].concat.apply [], _.pluck values, 'value'
-      @collection.reset objects
+      objects = _(values).chain().pluck('value')
+      if objects.first().has('results').value() # For articles wrapped in results
```

Comment:
`objects` can't be `null` based on the line above, right? in case it can, is `first()` safe to call on `null`?